### PR TITLE
HTML rendering of document

### PR DESCRIPTION
**Pull request by SteveLLamb**
Created from template (which was also updated), which AG-01 and AG-03 was created from. 

Since there was already info on this repo, and there is updates to the README files, I figured a PR was a good idea.

### CODE REVIEW

**Comment by SteveLLamb** [comment]
Note, once approved, will merge and turn on pages rendering.

**Comment by SteveLLamb** [comment]
@palemieux @fschleich, OK, I think this repo is at a point that it's ready to review for merge.

**Review by palemieux — CHANGES_REQUESTED** [review]
We need to figure out how to deal with abbreviated terms `<abbr>`

File: index.html [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+  </section>
+
+  <section class="annex" id="sec-process-flowchart">
+    <h2>Process Flowchart (Informative)</h2>
+
+    <figure id="figure-process-flowchart">
+      <figcaption>Example of the <a>Public CD</a> Process</figcaption>
+      <img class="figure" src="figure-process-flowchart.svg"></img>
+      <p class="note">A second Public CD (PCD2) is made available after addressing comments received on the first Public CD (PCD1).</p>
```
Comment (fschleich) [inline index.html]: This is supposed to be part of the figure caption.

**Review by fschleich — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (SteveLLamb) [reply index.html]: I updated it slightly for better reading in https://github.com/SMPTE/ag-22/pull/3/commits/497b350fccb5b6a443d0d0c3f0fc443c77e1d6f5

**Comment by SteveLLamb** [comment]
> We need to figure out how to deal with abbreviated terms `<abbr>`

@palemieux this is now resolved with new tooling in https://github.com/SMPTE/ag-22/pull/3/commits/52e2765d576711f386f7260bf99309041d9976dd

File: index.html (hunk 1 above)
Comment (SteveLLamb) [reply index.html]: this is now resolved with new tooling in https://github.com/SMPTE/ag-22/commit/52e2765d576711f386f7260bf99309041d9976dd

**Review by fschleich — APPROVED** [review]
lgtm

**Review by palemieux — APPROVED** [review]

**Pull request merged**